benchmarks/random.ts

```diff
@@ -1,32 +1,32 @@
 // https://github.com/ethereum/wiki/wiki/Benchmarks
-'use strict'
-import { keccak256 } from 'ethereumjs-util'
-import { BaseTrie } from '../dist'
+const crypto = require('crypto')
+import { CheckpointTrie } from '../dist'
 
-const ROUNDS = 1000
+const ROUNDS = 50000
 const SYMMETRIC = true
-const ERA_SIZE = 1000
+const ERA_SIZE = 10000
```

Comment:
I was also already close to just remove this since I didn't see the effect of this code part. Then likely do on your update. Would also be good if you can align the terminology of the two benchmarks (rounds, samples, runs, whatever 😄 ).

Reply:
Have updated.